src/commands/lightning/preview/app.ts

```diff
@@ -116,22 +192,116 @@ export default class LightningPreviewApp extends SfCommand<void> {
     //
     // Here we've chosen the second approach
     const idx = this.argv.findIndex((item) => item.toLowerCase() === '-o' || item.toLowerCase() === '--target-org');
+    let targetOrg: string | undefined;
     if (idx >= 0 && idx < this.argv.length - 1) {
-      orgOpenCommandArgs.push('--target-org', this.argv[idx + 1]);
+      targetOrg = this.argv[idx + 1];
     }
 
-    await this.config.runCommand('org:open', orgOpenCommandArgs);
+    const launchArguments = PreviewUtils.generateDesktopPreviewLaunchArguments(ldpServerUrl, appId, targetOrg);
+
+    await this.config.runCommand('org:open', launchArguments);
```

Comment:
I don't know if it's possible to use the new `sf` topic accessors in the command string (i.e. `org open`), but if so that would be preferred. The `<topic>:<sub-topic1>:<sub-topic2>` nomenclature is on the way out.

Reply:
Unfortunately that doesn't work and instead I see this printed to console:
![Screenshot 2024-06-10 at 11 19 51 AM](https://github.com/salesforcecli/plugin-lightning-dev/assets/65030660/c2c2253b-2ddd-46ed-a820-e28e3e2a451c)

Which doesn't make sense b/c even the error message prints the exact same command but still doesn't recognize it if I use `org open` instead of `org:open` Maybe that's why in [this thread](https://salesforce-internal.slack.com/archives/C06V045BZD0/p1715186066569749?thread_ts=1715185653.113019&cid=C06V045BZD0) they said to use `org:open`

Reply:
Sounds good. I didn't know if it would work or not, but it sounds like no, and probably doesn't bear further investigation than that at this point. I'm fine with it how it is.